src/main/java/com/example/rest/Employee.java

```diff
@@ -3,12 +3,10 @@
 import lombok.Builder;
 import lombok.Data;
 
-import java.math.BigInteger;
-
 @Data
 @Builder
 public class Employee {
     private String name;
     private Long id;
-    private BigInteger salary;
+    private Double salary;
```

Comment:
For money in java we have to use `BigDecimal` 
http://www.javapractices.com/topic/TopicAction.do?Id=213
http://www.javapractices.com/topic/TopicAction.do?Id=13